### ISSUE TITLE
Change default marker

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#192]
## Description
* The default circular marker gives the impression to select a region: unintuitive

## Acceptance criteria
* Change marker to be point specific

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Lucas-Nan** [comment]
![Image](https://github.com/amosproj/amos2024ss04-building-information-enhancer/assets/166795065/5a20890d-3ef9-4a94-a0aa-2a28ccac9135)

marker changed. trivial to adjust
